postgres/stack.yml

```diff
@@ -1,5 +1,4 @@
 # Use postgres/example user/password credentials
-version: '3.1'
```

Comment:
I believe this is still correct and needed for users of `docker-compose` v1 and docker swarm and doesn't hurt `docker-compose` v2 users.

Reply:
Reverted back.